# List of sequences to evaluate
sequence_id: ["005", "011", "016", "030", "061", "062", "069", "078", "086", "096", "202", "223", "225", "231", "249", "276", "286", "294", "322", "521"]
use_nyu_class: true
use_refined_id: false

compute_per_class_eval: true

ground_axis: 1
if_verbose: false

# 20 sequences
# ["005", "011", "016", "030", "061", "062", "069", "078", "086", "096", "202", "223", "225", "231", "249", "276", "286", "294", "322", "521"]

# 8 sequence
# ["011", "016", "030", "061", "078", "086", "096", "223"]
# ["061", "086", "096", "223", "225", "231", "249", "322"]

